app/controllers/exchangers_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ExchangersController < ApplicationController
```

Comment:
Are you sure you want the API and the normal use of the WebAPI to be at the same endpoint?

Reply:
For sure in case of real world Application  I would implement sub directory called `api` where we will store our endpoints for Restfull json api, but in our case since it's test task I think it's okay.